### PR TITLE
Data export new concept

### PR DESCRIPTION
**Pull request by DidierRLopes**
@jmaslek can you have a look at the concept of exporting. What do you think?

We could add this piece of code to the bottom of all commands, so that all data could be exported from an user perspective.

### CODE REVIEW

File: gamestonk_terminal/helper_funcs.py [hunk 1]
```diff
@@ -608,3 +608,28 @@ def find_tickers(submission):
             l_tickers_found.append(s_ticker.strip())
 
     return l_tickers_found
+
+
+def export_data(export_type, dir_path, func_name, df):
+
```
Comment (jmaslek) [inline gamestonk_terminal/helper_funcs.py:614]: I FINALLY GOT YOU.  WHERE ARE THE DOCSTRINGS

**Review by jmaslek — COMMENTED** [review]
So I think its a good idea.  We can add as needed.

You should also add "xls" as a choice (I forget the specifics of the differences, but both should point to df.to_excel).  Could also add a .pkl option or an .hdf option, but I feel that those would never get used.

File: gamestonk_terminal/helper_funcs.py (hunk 1 above)
Comment (DidierRLopes) [reply gamestonk_terminal/helper_funcs.py:614]: > I FINALLY GOT YOU. WHERE ARE THE DOCSTRINGS

To be fair, this was a concept PR :D

**Comment by DidierRLopes** [comment]
> So I think its a good idea. We can add as needed.
> 
> You should also add "xls" as a choice (I forget the specifics of the differences, but both should point to df.to_excel). Could also add a .pkl option or an .hdf option, but I feel that those would never get used.

Yea, not sure it would be used. I can do the xls yes!

**Comment by DidierRLopes** [comment]
> > So I think its a good idea. We can add as needed.
> > You should also add "xls" as a choice (I forget the specifics of the differences, but both should point to df.to_excel). Could also add a .pkl option or an .hdf option, but I feel that those would never get used.
> 
> Yea, not sure it would be used. I can do the xls yes!

I've tried this and can't use the to_excel() without installing another module. Let's leave it like this for now, and if a user requests xls, we can look into it.

Also, csv can be open as excel file.

**Review by jmaslek — APPROVED** [review]

**Pull request merged**